src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
+    private int supply;
+    private int buy;
+    private boolean isParsed = false;
+
+    private int getResult() {
+        return supply - buy;
+    }
+
```

Comment:
this code is totally redundant - you don't need any class level variables or methods to obtain the result

Reply:
![image](https://github.com/mate-academy/jv-work-with-file/assets/141054191/3a92e73b-3ca3-44d5-b592-6b2e310192ee)
This snippet I made to avoid this test failure, how to make this in another way?